docs/ui-coverage/configuration/elementgroups.mdx

```diff
@@ -152,3 +150,71 @@ A human-readable name for the group, shown in the elements list.
 ```
 Animal Option (4 instances)
 ```
+
+### Groups dynamic elements
+
+#### Config
+
+```json
+{
+  "uiCoverage": {
+    "elementGroups": [
+      {
+        "selector": "[id^='product']"
+      }
+    ]
+  }
+}
+```
+
+#### HTML
+
+```xml
+<body>
+  <div id="product125">Product 1</div>
+  <div id="product514">Product 2</div>
+  <div id="product256">Product 3</div>
```

Comment:
`div` is not interactive. make it `button` or `<div tabindex="0"` or `<div role="button"` or something?